src/restconfig/pom.xml

```diff
@@ -52,6 +52,23 @@
       <artifactId>javax.servlet-api</artifactId>
       <scope>provided</scope>
     </dependency>
+
+    <!-- add for rest api doc -->
+    <dependency>
+      <groupId>io.springfox</groupId>
+      <artifactId>springfox-swagger2</artifactId>
+      <version>2.9.2</version>
+    </dependency>
+    <dependency>
+      <groupId>io.springfox</groupId>
+      <artifactId>springfox-swagger-ui</artifactId>
+      <version>2.9.2</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.9.7</version>
```

Comment:
This looks like a fun addition, does it work well? For contributions visible to end-users we ask for an addition to the user guide (in the appropriate location).

We use dependencyManagement in the root pom.xml to manage the version of jackson used.